polaris-react/src/components/Page/Page.tsx

```diff
@@ -15,17 +15,9 @@ export interface PageProps extends HeaderProps {
   fullWidth?: boolean;
   /** Decreases the maximum layout width. Intended for single-column layouts */
   narrowWidth?: boolean;
-  /** Displays a divider between the page header and the page content */
-  divider?: boolean;
```

Comment:
Removed this prop as the styles were modified with uplift in a previous [PR](https://github.com/Shopify/polaris/pull/9537) and the class is just overriding pre-uplift spacing back to 0.

Reply:
Added `divider` prop removal to tasklist [here](https://github.com/Shopify/polaris/issues/9914) 👍